include/language/en_us.lang.php

```diff
@@ -2439,6 +2439,7 @@
     'LBL_OK' => 'Ok',
 
     'LBL_COLUMNS_FILTER_HEADER_TITLE' => 'Choose columns',
```

Comment:
There is issue, it should be here word  "Filter"

Reply:
@Mausino  see 'LBL_FILTER_HEADER_TITLE' => 'Filter' . 'LBL_FILTER_HEADER_TITLE' is being used for the filter icon. The columns 'LBL_COLUMN_CHOSER' is used for the column filter.

Reply:
@Dillon-Brown thank you. yes it was my mistake. sorry